what4/src/What4/Protocol/SMTLib2/Syntax.hs

```diff
@@ -51,11 +51,17 @@ module What4.Protocol.SMTLib2.Syntax
   , getUnsatCore
   , getAbduct
   , getAbductNext
+    -- * SyGus
```

Comment:
Nit pick: this PR is inconsistent about referring to "SyGuS" (with two capitalized S's) versus "SyGus" (with only one capitalized S). I _think_ the former is the more common convention, so we should use that consistently, such as in the line above. (There are other occurrences of "SyGus" in the PR to which this comment also applies.)